webapp/sass/layout/_navigation.scss

```diff
@@ -41,6 +41,14 @@
             width: 43px;
             z-index: 5;
 
+            > span {
+                > svg {
```

Comment:
Can we try to use classes instead of just relying on the HTML tags in the future? It makes it hard to understand what this CSS does and it'll be harder to move components around if we need to